src/pages/workspace/categories/WorkspaceCategoriesSettingsPage.tsx

```diff
@@ -12,66 +10,66 @@ import useLocalize from '@hooks/useLocalize';
-                                            <Switch
-                                                isOn={policy?.requiresCategory ?? false}
-                                                accessibilityLabel={translate('workspace.categories.requiresCategory')}
-                                                onToggle={updateWorkspaceRequiresCategory}
-                                                disabled={!policy?.areCategoriesEnabled || !hasEnabledOptions}
-                                            />
-                                        </View>
+                    <ScreenWrapper
+                        includeSafeAreaPaddingBottom={false}
+                        style={[styles.defaultModalContainer]}
+                        testID={WorkspaceCategoriesSettingsPage.displayName}
+                    >
+                        <HeaderWithBackButton title={translate('common.settings')} />
+                        <View style={styles.flexGrow1}>
+                            <OfflineWithFeedback
+                                errors={policy?.errorFields?.requiresCategory}
+                                pendingAction={policy?.pendingFields?.requiresCategory}
+                                errorRowStyles={styles.mh5}
+                            >
+                                <View style={[styles.mt2, styles.mh4]}>
+                                    <View style={[styles.flexRow, styles.mb5, styles.mr2, styles.alignItemsCenter, styles.justifyContentBetween]}>
+                                        <Text style={[styles.textNormal, styles.colorMuted, styles.flexShrink1, styles.mr2]}>{translate('workspace.categories.requiresCategory')}</Text>
+                                        <Switch
+                                            isOn={policy?.requiresCategory ?? false}
+                                            accessibilityLabel={translate('workspace.categories.requiresCategory')}
+                                            onToggle={updateWorkspaceRequiresCategory}
+                                            disabled={!policy?.areCategoriesEnabled || !hasEnabledOptions}
+                                        />
                                     </View>
-                                </OfflineWithFeedback>
-                            </View>
-                        </ScreenWrapper>
-                    )}
+                                </View>
+                                {isConnectedToAccounting && (
+                                    <Text style={[styles.textNormal, styles.colorMuted]}>{`${translate('workspace.categories.importedFromAccountingSoftware')}${translate(
+                                        'workspace.accounting.qbo',
+                                    )}`}</Text>
+                                )}
```

Comment:
Remove the whole view and replace it with:


```js
                        <ToggleSettingOptionRow
                            title={translate('workspace.categories.requiresCategory')}
                            subtitle={isConnectedToAccounting ? `${translate('workspace.categories.importedFromAccountingSoftware')} ${translate('workspace.accounting.qbo')}` : ''}
                            isActive={policy?.requiresCategory ?? false}
                            onToggle={updateWorkspaceRequiresCategory}
                            pendingAction={policy?.pendingFields?.requiresCategory}
                            disabled={!policy?.areCategoriesEnabled || !hasEnabledOptions}
                            wrapperStyle={[styles.mt2, styles.mh4]}
                            errors={ErrorUtils.getLatestErrorField(policy ?? {}, 'requiresCategory')}
                            onCloseError={() => Policy.clearPolicyErrorField(policy?.id ?? '', 'requiresCategory')}
                        />
```

Reply:
Also check the subtitle

Reply:
https://github.com/Expensify/App/pull/40942/commits/c9cec18f4ce7753f42a05b058dcd139abbb7728c